blocks/edit/prose/plugins/keyHandlers.js

```diff
@@ -47,6 +48,34 @@ export function getURLInputRule() {
   );
 }
 
+/**
+ * Converts "* " or "- " at the start of a line into a bullet list.
+ *
+ * @param {Schema} schema - The ProseMirror schema
+ * @returns {InputRule|null} the bullet list input rule, or null if
+ *   bullet_list node doesn't exist
+ */
```

Comment:
nit - can simplify comment to just 
`// Converts "* " or "- " at the start of a line into a bullet list.`

same for `getOrderedListInputRule`

We don't minify the js files, sotry to keep size down wherever possible.